lib/routes/causes.js

```diff
@@ -0,0 +1,56 @@
+const router = require('koa-router')()
+const models = require('../models')
+
+router
+  .get('/causes', async (ctx) => {
+    ctx.body = await models.db.cause.findAll({order: 'name ASC'})
+  })
+  .get('/causes/:id', async (ctx) => {
+    ctx.body = await models.db.cause.findById(ctx.params.id)
+  })
+  .post('/causes', async (ctx) => {
```

Comment:
I think that we will need to make this privileged as well.  This is not something that most users will be doing, this is an admin only operation.